packages/pwa-kit-extension-sdk/src/configs/webpack/overrides-resolver-loader.test.ts

```diff
@@ -51,12 +51,13 @@ describe('Overrides Resolver Loader', () => {
                     // Overrides
 
                     // Local project with overrides
-                    '/app/overrides/pages/sample-page.jsx': '// Base Project - Sample Page',
+                    '/app/overrides/@salesforce/extension-this/pages/sample-page.jsx':
+                        '// Base Project - Sample Page',
 
                     // Extensions with overrides
-                    '/node_modules/@salesforce/extension-this/src/overrides/pages/sample-page.jsx':
+                    '/node_modules/@salesforce/extension-this/src/overrides/@salesforce/this/pages/sample-page.jsx':
                         '// @salesforce/extension-this',
```

Comment:
The other extensions that try to override, in this case, are `extension-that` and `extension-other`.
```suggestion
                    '/node_modules/@salesforce/extension-that/src/overrides/@salesforce/extension-this/pages/sample-page.jsx':
                        '// @salesforce/extension-that',
```